plugins/Goals/Reports/Get.php

```diff
@@ -182,6 +184,17 @@ public function configureView(ViewDataTable $view)
                 }
             }
         } else if ($view->isViewDataTableId(Evolution::ID)) {
+            if (!empty($idSite) && Piwik::isUserHasAdminAccess($idSite)) {
+                $view->config->title_edit_entity_url = 'index.php' . Url::getCurrentQueryStringWithParametersModified(array(
+                    'module' => 'Goals',
+                    'action' => 'manage',
```

Comment:
this is quite tricky currently, it would likely change date etc unless we refactor it quite a bit. Can you check on demo.piwik.org it goes to same page